SwissTransferCore/Utils/Date+Extension.swift

```diff
@@ -0,0 +1,123 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import STResources
+
+// TODO: - Move to InfomaniakCore ?
+extension Date {
+    var toString: String {
```

Comment:
`toString` feels very "generic" could rename it to something that is more representative of the return.

Also maybe we can use the more modern Swift formatters ?